scripts/test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+ 
+coverage run manage.py test
```

Comment:
I recommend that this is changed to `coverage run -m pytest -v` due to the issue highlighted [here](https://stackoverflow.com/questions/29449285/flask-testing-why-does-coverage-exclude-import-statements-and-decorators/29453476#29453476)

Reply:
Command has been changed.